webui/react/src/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Router } from 'react-router-dom';
+import { unstable_HistoryRouter as HistoryRouter } from 'react-router-dom';
```

Comment:
This code should be safe because I set `history` lib to be the same version as `react-router-dom`
https://reactrouter.com/docs/en/v6/routers/history-router
https://github.com/remix-run/react-router/blob/f16c5490dfa75f15dcfb86d2a981a7c58a9d1a33/package.json#L44

Reply:
**suggestion:** can you add this comment above this line just so people reading this part of the code can lookup why it's called `unstable_HistoryRouter`
```
/**
 * It's considered unstable until `react-router-dom` can detect
 * history version mismatches when supplying your own history.
 * https://reactrouter.com/docs/en/v6/routers/history-router
 */
```